poolmgr/gp.go

```diff
@@ -262,6 +264,31 @@ func (gp *GenericPool) specializePod(pod *v1.Pod, metadata *fission.Metadata) er
 		return errors.New(fmt.Sprintf("Error from fetcher: %v", resp.Status))
 	}
 
+	logReq := logger.LogRequest{
```

Comment:
Sorry I should have been more clear -- I meant the whole functionality of setting up the logger, not just the one statement (upto line 285)